### ISSUE TITLE
vmalert: `query` function in templating returns 'runtime error'

### ISSUE THREAD

**Issue by vandud** [VictoriaMetrics/VictoriaMetrics#4181]
### Describe the bug

When I use 'query' function in templating then vmalert returns error

For example:
I set flag `-external.alert.source='{{ query "sort_desc(process_resident_memory_bytes)"}}'`
And see in the log lines like this:
```
Apr 21 18:28:13 mon-vm-01 vmalert-prod[14563]: 2023-04-21T15:28:13.456Z        error        VictoriaMetrics/app/vmalert/main.go:290        can not exec source template errors(1): key "tpl", template "{{ query \"sort_desc(process_resident_memory_bytes)\"}}": error evaluating annotation template: template: :1:241: executing "" at <query "sort_desc(process_resident_memory_bytes)">: error calling query: runtime error: invalid memory address or nil pointer dereference
```

### To Reproduce

Set flag `-external.alert.source='{{ query "sort_desc(process_resident_memory_bytes)"}}'`
Log output is below

### Version

v1.90.0

### Logs

Apr 21 18:28:13 mon-vm-01 vmalert-prod[14563]: 2023-04-21T15:28:13.456Z        error        VictoriaMetrics/app/vmalert/main.go:290        can not exec source template errors(1): key "tpl", template "{{ query \"sort_desc(process_resident_memory_bytes)\"}}": error evaluating annotation template: template: :1:241: executing "" at <query "sort_desc(process_resident_memory_bytes)">: error calling query: runtime error: invalid memory address or nil pointer dereference

**Comment by valyala** [comment]
The issue should be fixed in the commit 89a1c941c2bdb89e6508c7a73ff3661543e4e613 . This commit will be included in the next release of `vmalert`. In the mean time it is possible to build `vmalert` from this commit according to [these docs](https://docs.victoriametrics.com/vmalert.html#how-to-build-from-sources) and verify whether the issue is properly fixed there.

**Comment by valyala** [comment]
The bugfix for the issue has been included in [vmalert v1.91.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing the issue as fixed then.
